srunner/scenarios/actor_flow.py

```diff
@@ -319,3 +328,93 @@ def __del__(self):
         Remove all actors and traffic lights upon deletion
         """
         self.remove_all_actors()
+
+class CleanCrossActorFlow(BasicScenario):
```

Comment:
This needs a more clear name to understand the expected usecase of the scenario. Something like FastHighwayExit? Does this work if the exit is on the right? If so, then just HighwayExit?

Reply:
It works if the exit is on the right. HighwayExit is fine.